### PR TITLE
people/damiri: Change standing due to graduation

### PR DESCRIPTION
**Pull request by dcallega**
Update the personal page metadata accordingly.

### CODE REVIEW

File: content/people/damiri.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 name:
   first: Delaram
   last: Amiri
-role: student-phd
+role: alumnus-phd
 status: Ph.D. Candidate
```
Comment (igorburago) [inline content/people/damiri.md]: Please change the status to ‘Ph.D.’ (i.e., remove the ‘Candidate’ part).

**Review by igorburago — CHANGES_REQUESTED** [review]
After updating the `status` field, squash it back into a single commit
(removing the period at the end of the commit message and prepending
it with the `people/damiri:` prefix).

**Comment by igorburago** [comment]
I skimmed through the other two pull requests of yours, and would like
to ask you to adjust your commit message style slightly in all of them
to match the one generally established in the Git community.
Specifically, please:

- **Do not end the commit subject line with a period.**
- **Formulate the commit subject imperatively,** so that, prefaced
with “Apply this commit to”, it reads as a correct English sentence
(ignoring any colon-separated technical prefixes).

For more advice on commit message styling, I would recommend
taking a look at the “[How to Write a Git Commit Message][hgc]” article
by Chris Beams. It is from 2014, but all of the recommendations
in it still stand and are worth following.

[hgc]: https://chris.beams.io/posts/git-commit/

Speaking of imperative commit subjects, I also try to:

- **Formulate pull request titles imperatively**, e.g., “*Sort records
consistently*” (so it can be read as “[Merge this pull request to]
*sort records consistently*”).
- **Formulate issue titles descriptively**, e.g., “*The order of records
is inconsistent*” or “*Inconsistent ordering of records*” (so they can
be read as “[The issue is that] *the order of records is inconsistent*”
and “[The issue is due to] *inconsistent ordering of records*”,
respectively).

**Comment by igorburago** [comment]
Also, are the title of the dissertation and the month of the defense
(the one written at the bottom of the dissertation’s title page) known?

If so, it would be great to add a BibTeX entry for her dissertation
to the `static/bib/damiri.bib` file and list it in the publications
section on her page (next to the Master’s thesis).

(We may do that in a separate pull request later, if it is too much
of a hustle at the moment.)

**Comment by dcallega** [comment]
Thank you so much for the feedback! I did my best and I think I fixed this.
Does the message make more sense now?

**Comment by igorburago** [comment]
LGTM.

I assume, we leave adding the dissertation reference for later?

**Review by igorburago — APPROVED** [review]

**Comment by iburago** [comment]
Oh, I just realized that we had also forgotten to add an entry for
this newly acquired degree in the education section. Could you
please add the following after line 16?

```
- degree: Ph.D. in Computer Science
  institution: University of California, Irvine, U.S.
  year: 2020
```

**Review by igorburago — CHANGES_REQUESTED** [review]
Oh, I just realized that we have forgotten to add an entry for this
newly acquired degree in the education section. Could you please
add the following after line 16?

```
- degree: Ph.D. in Computer Science
  institution: University of California, Irvine, U.S.
  year: 2020
```

**Comment by dcallega** [comment]
I did not add it because I did not get the information from Delaram about her thesis' title.
However it makes sense to add the degree as is and update that later.

**Comment by igorburago** [comment]
> I did not add it because I did not get the information from Delaram
> about her thesis' title.

That’s fine.

> However it makes sense to add the degree as is and update that later.

Of course. (In fact, it would perhaps be great to do that for all alumni.)

**Comment by igorburago** [comment]
LGTM.

**Review by igorburago — APPROVED** [review]

**Pull request merged**